src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -534,14 +559,38 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
   }
 
   deleteFile(pathAndName: string): void {
-    this.ussSrv.deleteFile(pathAndName)
+    let deleteSubscription = this.ussSrv.deleteFile(pathAndName)
     .subscribe(
       resp => {
+        this.sendNotification('Editor', 'Deleted: ' + pathAndName);
         this.updateUss(this.path);
       },
-      error => this.errorMessage = <any>error
+      error => {
+        if (error.status == '403') { //Forbidden (TODO: Implement ZSS functionality to return a 403 for permissions problems)
+          this.sendNotification('Editor', 'Incorrect permissions to delete: ' + pathAndName);
+        } else if (error.status == '410') { //Gone (TODO: Implement ZSS functionality to return a 410 for a file/folder that has already been deleted)
+          this.sendNotification('Editor', 'Failed to delete: ' + pathAndName);
+        } else if (error.status == '500') { //Internal Server Error
+          this.sendNotification('Editor', 'Failed to delete: ' + pathAndName);
+        } else {
+          this.sendNotification('Editor', 'Uknown error occured for: ' + pathAndName);
+        }
+        this.errorMessage = <any>error }
     );
+    setTimeout(() => {
+      if (deleteSubscription.closed == false) {
+        this.snackBar.open('Deleting ' + pathAndName + '... Larger payloads may take longer. Please be patient.', 
+          'Dismiss', { duration: 5000,   panelClass: 'center' });
+      }
+    }, 4000);
   }
+
+  sendNotification(title: string, message: string): number {
+    let pluginId = this.pluginDefinition.getBasePlugin().getIdentifier();
+    let notification = new ZoweNotification(title, message, 1, pluginId, "org_zowe_zlux_editor_snackbar");
```

Comment:
Please don't use a theme, but perhaps comment the use of it out, so we can show people how they might use a theme in the future.